src/rnp/fficli.cpp

```diff
@@ -1552,6 +1552,13 @@ rnp_cfg_set_ks_info(rnp_cfg &cfg)
         defhomedir = true;
     }
 
+    struct stat st;
+
+    if (rnp_stat(homedir.c_str(), &st) || rnp_access(homedir.c_str(), R_OK | W_OK)) {
+        ERR_MSG("Home directory '%s' does not exist or is not writeable!", homedir.c_str());
```

Comment:
Looks like 'writable' is more common spelling then 'writeable'.

Reply:
Ping, thanks @antonsviridenko !